main.py

```diff
@@ -38,7 +39,22 @@ def main():
         config_file = networkconfig[conf]
         with tf.Session() as sess:
             builder = ModelBuilder(config_file, sess)
+            builder.add_input_layer()
+
+            # Add a number of hidden layers
```

Comment:
This is more modular, if someone want to set up the network in another way than how it is done in the build method then there is no way of building the model